### PR TITLE
refactor: Format code, use structs

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/DssGov.sol [hunk 1]
```diff
@@ -46,121 +45,118 @@ contract DssGov {
...
+    event    AddProposer(address indexed usr);
     event RemoveProposer(address indexed usr);
-    event MintGas(bytes32 indexed src, address indexed usr, uint256 amt);
-    event BurnGas(bytes32 indexed src, address indexed usr, uint256 amt);
-    event Delegate(address indexed owner, address indexed newDelegate);
-    event Lock(address indexed usr, uint256 wad);
-    event Free(address indexed usr, uint256 wad);
-    event Clear(address indexed usr);
-    event Ping(address indexed usr);
+    event        MintGas(bytes32 indexed src, address indexed usr, uint256 amt);
```
Comment (gbalabasquer) [inline src/DssGov.sol:115]: Hmmm not really sure I like this format

Comment (lucas-manuel) [reply src/DssGov.sol:115]: No worries Ill change it back

Comment (gbalabasquer) [reply src/DssGov.sol]: Formatting is something we need to discuss how to standardize. I think I'm not even being myself consistent with the formatting everywhere. So we should probably set some rules.

Comment (lucas-manuel) [reply src/DssGov.sol]: I think that's a great idea.

**Review by gbalabasquer — APPROVED** [review]
LGTM. It can be squashed and merged.

**Pull request merged**